### PR TITLE
Created avatar and alert components

### PR DESCRIPTION
**Pull request by deekshithmd**
Avatar and Alert components created, reviews invited for the improvement

### CODE REVIEW

**Comment by ashishkatiyar2404** [comment]
your code looks good... 
clean and well structured

PS:- you can use flex-box instead of using float(I have seen in one line). Just a suggestion.

File: components/alerts/alerts.css [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+}
+.alert-warning-close{
+    display: flex;
+    align-items: center;
+}
+.btn-close{
+    font-size: 4rem;
+    border: none;
+}
+#alert-warning-close-btn{
```
Comment (hsnice16) [inline components/alerts/alerts.css]: Don't use Id selector for CL. Always try to use class selector.

File: components/alerts/alerts.css [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+}
+.alert-box-border{
+    border: 0.3rem solid black;
+}
+.alert-primary{
+    background-color: rgb(177, 237, 241);
+    color: rgb(42, 139, 219);
+}
+.alert-secondary{
+    background-color: rgb(210, 238, 235);
```
Comment (hsnice16) [inline components/alerts/alerts.css]: Try to use CSS variable for color, I think they have mentioned in FAQ too (on LMS).

File: components/alerts/alerts.css [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+    color: rgb(71, 62, 62);
+}
+.alert-success{
```
Comment (hsnice16) [inline components/alerts/alerts.css:24]: Give atleast a line break before starting new Class styling.

**Review by hsnice16 — CHANGES_REQUESTED** [review]

**Comment by deekshithmd** [comment]
> your code looks good... clean and well structured
> 
> PS:- you can use flex-box instead of using float(I have seen in one line). Just a suggestion.

ok,Thank you

File: components/alerts/alerts.css (hunk 1 above)
Comment (deekshithmd) [reply components/alerts/alerts.css]: ok, I will do that

File: components/alerts/alerts.css (hunk 2 above)
Comment (deekshithmd) [reply components/alerts/alerts.css]: ok, I will implement that

File: components/alerts/alerts.css (hunk 3 above)
Comment (deekshithmd) [reply components/alerts/alerts.css:24]: ok, Thank you

**Pull request merged**